packages/blockchain-wallet-v4-frontend/src/data/coins/accountTypes/accountTypes.classes.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable max-classes-per-file */
+import { CallEffect, SelectEffect } from 'redux-saga/effects'
+
+import { APIType } from '@core/network/api'
+import { CoinType, RemoteDataType } from '@core/types'
+import { RootState } from 'data/rootReducer'
+import { RequestExtrasType, SwapAccountType } from 'data/types'
+
+export enum CoinAccountTypeEnum {
+  CUSTODIAL = 'CUSTODIAL',
+  DYNAMIC_SELF_CUSTODY = 'DYNAMIC_SELF_CUSTODY',
+  ERC20 = 'ERC20',
+  NON_CUSTODIAL = 'NON_CUSTODIAL'
+}
+
+export class AccountTypeClass {
```

Comment:
That should not really be a class. Just an interface.

Same for the one below.